pages/k8s/logging.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Logging | Canonical Distribution of Kubernetes&reg;"
+keywords: juju, logging, debug-log
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: logging.html
+layout: base
+toc: False
+summary: This page is currently a work in progress. For existing documentation, please visit <a href="https://kubernetes.io/docs/getting-started-guides/ubuntu/"> https://kubernetes.io/docs/getting-started-guides/ubuntu/ </a>
+---
+
+# Logging
+
+<div class="p-notification--information">
+  <p class="p-notification__response">
+    <span class="p-notification__status">Note:</span>
+This documentation assumes you are using version 2.4.0 or later of
+<strong>Juju</strong> . If you are using an earlier version you should
+check the  <a href="https://docs.jujucharms.com/stable/en/troubleshooting-logs">
+ relevant <emphasis>Juju</emphasis> documentation </a> as some of the associated
+ commands have changed.
+  </p>
+</div>
+
+
+Your **CDK** deployment has centralised logging set up as default. Each unit in
```

Comment:
This is really only true for the juju log, not the kubernetes stuff. It implies to me that I can see all logging from my cluster in a central place.

Reply:
Yes, I agree. I will reword this bit, and probably add an issue to add some info about kubernetes logs (that isn't in the upstream docs currently)

Reply:
We could also include logging stuff from https://medium.com/@kwmonroe/monitor-your-kubernetes-cluster-a856d2603ec3n